.circleci/config.yml

```diff
@@ -47,12 +47,19 @@ install_dep: &install_dep
         python -c 'import torch; print("Torch version:", torch.__version__)'
         python -m torch.utils.collect_env
 
-install_repo: &install_repo
+install_repo_cpu: &install_repo_cpu
   - run:
       name: Install Repository
       command: |
         python setup.py build develop
 
+install_repo_gpu: &install_repo_gpu
+  - run:
+      name: Install Repository
+      command: |
+        export CUDA_HOME=/usr/local/cuda-10.1
+        python setup.py --fused-adam build develop
```

Comment:
Maybe instead of an option we can do something like PyTorch3D inside setup.py:

force_cuda = os.getenv("FORCE_CUDA", "0") == "1"
    if (torch.cuda.is_available() and CUDA_HOME is not None) or force_cuda:

https://github.com/facebookresearch/pytorch3d/blob/master/setup.py

Reply:
Changed!